app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizard.js

```diff
@@ -2,7 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { noop, Modal, Wizard, Icon, Button } from 'patternfly-react';
 import { connect } from 'react-redux';
-import { find } from 'lodash';
+
+const reduxFormMap = {
```

Comment:
I think we should pass this in as a prop, if we must have it in `ModalWizard`. I want to try and avoid anything specific to V2V in `ModalWizard.js` if possible, as it is a candidate component for being moved to `patternfly-react`.